src/Entity/Geo/ImpreciseGeoCoordinates.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Justimmo\Api\Entity\Geo;
+
+use Justimmo\Api\Annotation as JUSTIMMO;
+
+/**
+ * @JUSTIMMO\Entity()
+ */
+class ImpreciseGeoCoordinates extends GeoCoordinates
+{
+    public const TYPE_FUZZY   = 'fuzzy';
+    public const TYPE_PLZ_ORT = 'plz_ort';
+    public const TYPE_INVALID = 'invalid';
```

Comment:
Ist invalid ein eigener Punkt mit geokoordinaten? Falls nein sollte die klasse nicht in einem invaliden zustand instanziert werden können?